scripts/bridgeETH.ts

```diff
@@ -45,39 +59,50 @@ const FUEL_MESSAGE_TIMEOUT_MS = 60_000;
 
   // wait for message to appear in fuel client
   console.log('Waiting for ETH to arrive on Fuel...');
-  const depositMessage = await fuels_waitForMessage(env.fuel.provider, fuelAccount.address, depositMessageNonce, FUEL_MESSAGE_TIMEOUT_MS);
+  const depositMessage = await fuels_waitForMessage(
+    env.fuel.provider,
+    fuelAccount.address,
+    depositMessageNonce,
+    FUEL_MESSAGE_TIMEOUT_MS
+  );
   if (depositMessage == null)
-    throw new Error('message took longer than ' + FUEL_MESSAGE_TIMEOUT_MS + 'ms to arrive on Fuel');
+    throw new Error(`message took longer than ${FUEL_MESSAGE_TIMEOUT_MS}ms to arrive on Fuel`);
   console.log('');
 
   // the sent ETH is now spendable on Fuel
   console.log('ETH was bridged to Fuel successfully!!');
 
   // note balances of both accounts after transfer
   console.log('Account balances:');
-  console.log('  Ethereum - ' + formatEther(await ethereumAccount.getBalance()) + ' ETH (' + ethereumAccountAddress + ')');
-  console.log('  Fuel - ' + fuels_formatEther(await fuelAccount.getBalance(ZeroBytes32)) + ' ETH (' + fuelAccountAddress + ')');
+  console.log(`  Ethereum - ${formatEther(await ethereumAccount.getBalance())} ETH (${ethereumAccountAddress})`);
+  console.log(`  Fuel - ${fuels_formatEther(await fuelAccount.getBalance(ZeroBytes32))} ETH (${fuelAccountAddress})`);
   console.log('');
 
   // TODO: the below step is only there to avoid a bug in the SDK when a wallet has spendable messages
+  /*
   console.log('[BUG FIX] Converting message input to coin input...');
-  const fMessageToCoinTx = await fuels_messageToCoin(fuelAccount, depositMessage);
+  const fMessageToCoinTx = await fuels_messageToCoin(fuelAccount, depositMessage, fuelTxParams);
   const fMessageToCoinTxResult = await fMessageToCoinTx.waitForResult();
   if (fMessageToCoinTxResult.status.type !== 'success') {
     console.log(fMessageToCoinTxResult);
     throw new Error('failed to convert message to coin');
   }
-  console.log('  Ethereum - ' + formatEther(await ethereumAccount.getBalance()) + ' ETH (' + ethereumAccountAddress + ')');
-  console.log('  Fuel - ' + fuels_formatEther(await fuelAccount.getBalance(ZeroBytes32)) + ' ETH (' + fuelAccountAddress + ')');
+  console.log(`  Ethereum - ${formatEther(await ethereumAccount.getBalance())} ETH (${ethereumAccountAddress})`);
+  console.log(`  Fuel - ${fuels_formatEther(await fuelAccount.getBalance(ZeroBytes32))} ETH (${fuelAccountAddress})`);
   console.log('');
+  */
```

Comment:
Should this be deleted?

Reply:
Yes, the latest SDK updates fixed this issue